### PR TITLE
Added a new project: i-can-not-react

### PR DESCRIPTION
**Pull request by KirankumarAmbati**
i-can-not-react is an Open Source project started to help new devs to start learning Reactjs with ease.

### CODE REVIEW

**Comment by KirankumarAmbati** [comment]
**CC:** @ritwik12

**Comment by ritwik12** [comment]
@KirankumarAmbati Hey, sorry for the delay. Let me check why it is failing.

File: _data/projects/i-can-not-react.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: i-can-not-react
+desc: "The Complete Book on Reactjs"
+site: "https://github.com/KirankumarAmbati/I-can-not-REACT"
```
Comment (ritwik12) [inline _data/projects/i-can-not-react.yml]: @KirankumarAmbati I guess the reason for failing travis are these '"'. Please remove these from line 2 and line 3 both.

File: _data/projects/i-can-not-react.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- react
+- reactjs
+- javascript
+- javascript-framework
+- es6
+upforgrabs:
+  name: good first issue
+  link: "https://github.com/KirankumarAmbati/I-can-not-REACT/issues?q=is%3Aissue+is%3Aopen+label%3A"good+first+issue"
```
Comment (ritwik12) [inline _data/projects/i-can-not-react.yml]: @KirankumarAmbati Please change this to https://github.com/KirankumarAmbati/I-can-not-REACT/labels/good%20first%20issue

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by KirankumarAmbati** [comment]
Hi @ritwik12,

Made necessary changes. Please review them ! Thank you !!

File: _data/projects/i-can-not-react.yml (hunk 2 above)
Comment (ritwik12) [reply _data/projects/i-can-not-react.yml]: @KirankumarAmbati This is still pending. Please correct this.

Comment (KirankumarAmbati) [reply _data/projects/i-can-not-react.yml]: Done ! 👍 @ritwik12

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@KirankumarAmbati Sorry for the delay.

**Comment by ritwik12** [comment]
@KirankumarAmbati Thanks!!

**Pull request merged**